### DISCUSSION TITLE
Export 'forked from' and 'fork source' fields

### DISCUSSION THREAD

**Discussion by wolfenmark** [seart-group/ghs#22] [Ideas]
Hi,
I was wondering if it could be useful to have forked parameters like 'forked from' and 'fork source' (specifically for repos that are forks).

I know that every possible detail might be useful in one or more cases. Just wondering if these two fields are worth having in the output.

Example: if you need to analyze forks and fork chains starting from a subset of projects GHS could be used directly for that kind of research if fork chains could be reconstructed directly without scraping GH.

**Comment by emadpres** [comment]
Hi,

Adding a list of forked projects will cost more GitHub API calls which is something that already restricted us. So, probably this additional piece of info, while useful as you said in certain cases, will not make it into the platform, unless address the GitHub limits.

**Comment by wolfenmark** [comment]
Are you sure that it will cost extra API calls?
For example you have 'fork: [true | false]' information in the repo response (GET /repos/{owner}/{repo}).

I agree that it is not the most important information but I suspect that you already have it in the responses you get. So if API call number is the deciding factor you can investigate further if the info is already available.

Obviously using the dedicated fork API endpoints would be cleaner and more robust.
